src/renderer/components/settings-general-appearance.tsx

```diff
@@ -96,6 +106,11 @@ export class AppearanceSettings extends React.Component<
       this.setState({ themes, selectedTheme });
     });
 
+    autorun(async () => {
```

Comment:
What's the difference here when use `autorun`? It's something that I never see before so yeah, that looks strange for me.